app/components/channel_list/loading/index.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import LottieView from 'lottie-react-native';
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+
+const Loading = () => (
```

Comment:
can we add props to define the size on this component, I mean the size of the loading indicator and maybe a style of the container? I believe this loading indicator will be use in various places (FYI @jfrerich). @matthewbirtch can you confirm what I'm saying is true

Reply:
Ah there is already a `@components/loading` which is a wrapper for `<ActivityIndicator ...`, which is why I made this `@components/channel_list/loading` so I'm not sure where else it would be used. 

We can eventually replace the existing `<Loading...` but I thought for just this PR it would suffice.